react/components/molecules/tabs-text/tabs-text.js

```diff
@@ -45,13 +46,37 @@ export class TabsText extends PureComponent {
             animatedBarOffset: undefined
         };
         this.tabLayouts = {};
+        this.XScroll = 0;
+        this.overflows = false;
     }
 
     onTabPress = tabSelectedIndex => {
         this.setState({ tabSelected: tabSelectedIndex }, () => {
             this._updateBar();
             this.props.onTabChange(this.state.tabSelected);
         });
+        const deviceWidth = Dimensions.get("window").width;

```

Comment:
This should use a strategy that allows for the tabs not to use the whole width, otherwise it will break in scenarios such as:
- Being inside a container that isn't using the whole screen
- Having tabs with width 50%